app/api.py

```diff
@@ -12,6 +12,7 @@
 from app.model import MatcherSortSearch, MatcherSortSearchResource
 from app.vader_sentiment import vader_score
 from app.computer_assignment import computer_assignment_visualizer
+from app.schema import Mentor, MentorUpdate, Mentee, MenteeUpdate
 
 API = FastAPI(
     title='Underdog Devs DS API',
```

Comment:
Awsome team work. We should probably update the API version here to 0.45.4

Reply:
Done! Thanks for the heads up!